src/main/java/org/isf/medicals/model/Medical.java

```diff
@@ -21,6 +21,10 @@
  */
 package org.isf.medicals.model;
 
+import org.isf.medtype.model.MedicalType;
+import org.isf.utils.db.Auditable;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
```

Comment:
The import order defined in both Eclipse and IntelliJ has `jakarta` coming before `org`

Reply:
fixed, thanks!